### ISSUE TITLE
Explorer loses keyboard focus when opening temp view of file

### ISSUE THREAD

**Issue by mlepage** [Microsoft/vscode#41418]
I would like to be able to use the keyboard (arrow keys and enter) to view the content of a series of files (in the explorer) in succession.

This does not work beyond viewing the first file, because keyboard focus leaves the explorer. Is this intentional or a bug?

- VSCode Version: 1.19.1
- OS Version: Debian

Steps to Reproduce:

1. Open a folder (list of files) in explorer on left side.
2. Focus on a file in explorer (so it shows contents in temp view).
3. Now use arrow key to go down to another file in explorer.
4. Press enter to show that file in the temp view. So far so good.
5. Now we want to continue to down arrow key and enter key to view more files temporarily. This does not work, because focus has left the explorer and is now in the cursor/caret in the temp view.

**Comment by isidorn** [comment]
This is designed behavior.
Though we might except a PR that adds an option that whenever we open a file from a tree the focus is left in the tree. 'workbench.preserveFocusOnOpenFile' for example
This woudl then also apply for other trees like the debug call stack view, debug breakopints and so on.

@bpasero fyi
